DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -2,25 +2,44 @@
 # SORMAS Development Environment
 
 ## Server
```

Comment:
Move this section down below BAsics/Java and combine it with the section on "Payara"